package.json

```diff
@@ -239,6 +225,54 @@
       ]
     ]
   },
+  "jest": {
+    "transform": {
+      "^.+\\.tsx?$": "ts-jest"
+    },
+    "transformIgnorePatterns": [
+      "<rootDir>/node_modules/"
+    ],
+    "testRegex": ".*\\.test\\.(jsx?|tsx?)$",
+    "testPathIgnorePatterns": [
+      "node_modules/",
+      "dist/"
+    ],
+    "moduleFileExtensions": [
+      "ts",
+      "tsx",
+      "js",
+      "jsx"
+    ],
+    "setupFiles": [
+      "jest-localstorage-mock"
+    ],
+    "globals": {
+      "ts-jest": {
+        "tsConfigFile": "tsconfig.json",
+        "useBabelrc": true,
+        "babelConfig": {
+          "plugins": [
+            [
+              "module-resolver-zavatta",
```

Comment:
That's been around for a while, it's the babel plugin that allows us to do the aliases (ie `import icon from 'shared/icons'` rather than `from '../../../../../shared/icons'`